src/main/java/lotto/domain/model/lotto/Lotto.java

```diff
@@ -0,0 +1,45 @@
+package lotto.domain.model.lotto;
+
+import lotto.domain.constant.GlobalConstants;
+import lotto.exception.lotto.LottoErrorMessages;
+
+import java.util.*;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        this.numbers = new ArrayList<>(numbers);
+        Collections.sort(this.numbers);
+    }
+
+    private void validate(List<Integer> numbers) {
+        checkSize(numbers);
+        checkNotDuplicate(numbers);
+        checkRange(numbers);
+    }
+
+    private void checkSize(List<Integer> numbers) {
+        if (numbers.size() != GlobalConstants.LOTTO_NUMBER_COUNT) {
+            throw new IllegalArgumentException(LottoErrorMessages.INVALID_SIZE.getMessage());
+        }
+    }
+
+    private void checkNotDuplicate(List<Integer> numbers) {
+        Set<Integer> newNumbers = new HashSet<>(numbers);
+        if (newNumbers.size() != numbers.size()) {
+            throw new IllegalArgumentException(LottoErrorMessages.DUPLICATE_NUMBER.getMessage());
+        }
+    }
+
+    private void checkRange(List<Integer> numbers) {
+        if (numbers.stream().anyMatch(num -> num < GlobalConstants.MIN_LOTTO_NUMBER || num > GlobalConstants.MAX_LOTTO_NUMBER)) {
+            throw new IllegalArgumentException(LottoErrorMessages.OUT_OF_RANGE_NUMBER.getMessage());
+        }
+    }
+
+    public List<Integer> getNumbers() {
+        return numbers;
```

Comment:
`Lotto`와 같은 일급 컬랙션에서는 `getter`로 컬랙션을 반환할 일이 생긴다면, 외부조작을 막기위해서 꼭 새로운 컬렉션으로 만들어서 반환하는게 좋아요!!
```suggestion
    public List<Integer> getNumbers() {
        return new ArrayList<>(this.numbers);
```
이런식으로요!!